lisa/messages.py

```diff
@@ -52,8 +52,12 @@ class TransportProtocol(str, Enum):
     Udp = "UDP"
 
 
+class CommunityTestMessage(MessageBase):
```

Comment:
There should be a `TestResultMessageBase(MessageBase)`, and the CommunityTestMessage and TestResultMessage derive from it. Lisa use the TestResultMessage, but the community test cases dispatch the CommunityTestMessage.